spring-integration-core/src/main/java/org/springframework/integration/util/MessagingMethodInvokerHelper.java

```diff
@@ -979,7 +979,7 @@ private void setExclusiveTargetParameterType(TypeDescriptor targetParameterType,
 		}
 	}
 
-	private static class ParametersWrapper {
+	public static class ParametersWrapper {
```

Comment:
Thank you for such a good catch, @garyrussell ! 😄 

Really appreciate.

Previously this class wasn't `static`. 
But there is really nothing to make it tied with an outer class and the `static` is better for instance.

So, right now making it as a `public` (because we need it any way) makes me think that we should poll this class to the package level and bring it better public name. Just in case some one some where will be interested in similar abstraction.

WDYT?

Otherwise yes, LGTM.